src/Common/ItechArt.Common/ILogger.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+
+namespace ItechArt.Common
+{
+    public interface ILogger
+    {
+        void Write(LogLevel logLevel, string logMessage, Exception exception = null);
```

Comment:
1) make ILogger abstraction more clear
2) replace all 3rd party dependencies (LogLevel) with own instances
3) investigate different log levels' purposes, and specify exception param only where it's needed
4) let's reorder methods by log level ascending